lib/Digitick/Sepa/TransferFile/Facade/CustomerCreditFacade.php

```diff
@@ -34,6 +34,8 @@ public function addPaymentInfo($paymentName, array $paymentInformation)
             $paymentInformation['debtorName']
         );
 
+        $payment->setDueDate($this->HandleDueDate($paymentInformation));
```

Comment:
ähm .. you need to update the calls of the renamed method as well ...
Thinking longer about the name it should read createDueDateFromPaymentInformation this implies a creation of the date if it is not set on the payment information.